### PR TITLE
update tableClasses in public plugin 3rd PR

### PR DESCRIPTION
**Pull request by abhinandan13jan**
# Addresses
https://issues.redhat.com/browse/ODC-5816

# Screenshots

IngressCRDAlertM
# Cronjob & HPA
build config imagestream

cronjob hpa

# Tests
None changed

# Browser conformance

Chrome

### CODE REVIEW

File: frontend/public/components/ingress.jsx [hunk 1]
```diff
@@ -58,10 +58,10 @@ const TLSCert = ({ ingress }) => {
...
 
 const tableColumnClasses = [
-  classNames('col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-md-3', 'col-sm-4', 'col-xs-6'),
-  classNames('col-md-3', 'col-sm-4', 'hidden-xs'),
-  classNames('col-md-3', 'hidden-sm', 'hidden-xs'),
+  '',
+  '',
+  'pf-m-hidden pf-m-visible-on-md',
+  'pf-m-hidden pf-m-visible-on-md',
```
Comment (sg00dwin) [inline frontend/public/components/ingress.jsx:64]: Currently we show the `Hosts` column at browser sizes > 992px which is `pf-m-visible-on-lg`  
```suggestion
  'pf-m-hidden pf-m-visible-on-md',
  'pf-m-hidden pf-m-visible-on-lg',
```

Comment (abhinandan13jan) [reply frontend/public/components/ingress.jsx]: @sg00dwin Updated

**Review by sg00dwin — APPROVED** [review]

**Comment by karthikjeeyar** [comment]
Verified locally, works as expected
/lgtm

**Review by jerolimov — APPROVED** [review]
Looks good
/approve

**Pull request merged**